conf/messages.en

```diff
@@ -174,7 +175,7 @@ askSecondaryWarehouses.detailsLink = Why should I register a warehouse?
 askSecondaryWarehouses.detailsContent = You can delay the point you will have to report liable drinks in a return and pay the levy on them if you register the warehouses you use to store them.
 askSecondaryWarehouses.checkYourAnswersLabel = Do you want to register any of the UK warehouses you use to store liable drinks?
 askSecondaryWarehouses.error.required = Select yes if you want to register any of the UK warehouses you use to store liable drinks
-askSecondaryWarehouses.change.hidden = AskSecondaryWarehouses
+askSecondaryWarehouses.change.hidden = Ask Secondary Warehouses
```

Comment:
This hidden text comes following "Change" so "Change Ask Secondary Warehouses" does not make sense. It should be something like "the UK warehouses you use to store liable drinks"